deeprank2/query.py

```diff
@@ -183,19 +184,30 @@ def build(
         Returns:
             :class:`Graph`: The resulting :class:`Graph` object with all the features and targets.
         """
+        # TODO: Should this be an internal method, as it is only called upon by QueryCollection and not by user?
 
-        try:
-            feature_modules = [importlib.import_module('deeprank2.features.' + name) for name in feature_modules]
-        except TypeError:
-            if isinstance(feature_modules, ModuleType):
-                feature_modules = [feature_modules]
-            elif isinstance(feature_modules, str):
-                feature_modules = [importlib.import_module('deeprank2.features.' + feature_modules)]
+        # TODO: I would prefer if  `_set_feature_modules` would return a list of `ModuleType` instead of `str`,
+        # but this leads to an exception in the pool function in `QueryCollection.process`, that I don't know how to solve.
```

Comment:
This seems more for a separate issue, and needs more details about the exception